geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/rest/RebalanceManagementDunitTest.java

```diff
@@ -91,7 +92,7 @@ public void rebalance() throws Exception {
         .isGreaterThanOrEqualTo(cmr.getOperationStart().getTime());
     assertThat(result.getRebalanceRegionResults().size()).isEqualTo(2);
     RebalanceRegionResult firstRegionSummary = result.getRebalanceRegionResults().get(0);
-    assertThat(firstRegionSummary.getRegionName()).isIn("customers1", "customers2");
+    assertThat(firstRegionSummary.getRegionName()).isIn("/customers1", "/customers2");
   }
```

Comment:
REST management API does not plan to support sub-regions, so canonical form of region name is expected to be without the trailing /

Reply:
This should be fixed now.